README.md

```diff
@@ -1 +1,49 @@
-# 2021-2-SysArq-Doc
\ No newline at end of file
+
+![image](https://user-images.githubusercontent.com/78980992/157777487-6715c44a-bc49-4b61-a7ff-b54c4e6fda7f.png)
+
+
+### Entenda nossa proposta
+O SysArq surgiu da necessidade por um sistema de gerencimento de arquivos para a Polícia Civil do Goiás.
+<br>
+<br>
+Com isso, o SysArq busca digitalizar o processo de armazenar os dados dos documentos, além de auxiliar na localização dos arquivos armazenados e registrar suas trasitações. 
+
+### Conheça nossa equipe
+
+| Membro | E-mail | Github | Papel |Matricula|
+|-------------------------------|--------------------------|----------------------------------|----------------------|------------|
+| Iago Oliveira Monteiro Ribeiro | iagorag363@gmail.com	| [iagoomr](https://github.com/iagoomr) | EPS | 16/0124735 |
+| João Luis Takur Baraky Dias | jlbaraky@gmail.com	| [baraky](https://github.com/baraky) | EPS | 18/0033646 |
+| Rafael Santos Teodosio | rafzteodosio@gmail.com	| [rafaelteodosio](https://github.com/rafaelteodosio) | EPS | 16/0142466 |
+| Weiller Fernandes Pereira | weillerfernandes@gmail.com	| [WeillerFernandes](https://github.com/WeillerFernandes) | EPS | 13/0137995 |
+| Alex Gabriel Alves Faustino | alexgabriel3357@gmail.com	| [Gabriel-Axl](https://github.com/Gabriel-Axl) | MDS | 20/0056603 |
+| André Corrêa da Silva | andre.correa.silva203@gmail.com	| [dartmol203](https://github.com/dartmol203) | MDS | 20/0014447 |
+| Arthur Ferreira Rodrigues | fr.arthur.fr@gmail.com	| [ArthurFerreiraRodrigues](https://github.com/ArthurFerreiraRodrigues) | MDS | 20/0056981 |
+| Arthur Henrique Amaral Santana | arthurhenrir4@gmail.com	| [black258br](https://github.com/black258br) | MDS | 20/0035371 |
+| Cleber de Oliveira Brant | cleberobb@hotmail.com	| [Cleberbrant](https://github.com/Cleberbrant) | MDS | 20/0061216 |
+| Guilherme Puida Moreira | guilhermepmoreira00@gmail.com	| [guilherme-puida](https://github.com/guilherme-puida) | MDS | 20/0019015 |
+| Paulo Maciel Torres Filho | paulomacieltorresfilho@gmail.com	| [paulomacieltorresfilho](https://github.com/paulomacieltorresfilho) | MDS | 20/0025937 |
+| Wesley Carvalho Lira | wesleeylirac12@gmail.com	| [Weslin-0101](https://github.com/Weslin-0101) | MDS | 20/0044559 |
+
+### Conheça nossa documentação
+* [SysArq Docs](https://fga-eps-mds.github.io/2021.1-PC-GO1/) //ainda falta linkar a nova wiki
```

Comment:
Pode tirar esse comentário, deixe apenas ```SysArq Docs``` (sem link mesmo, cru). Quando a wiki for criada a gente atualiza o README com o link.

Reply:
comentário retirado